libr/core/cconfig.c

```diff
@@ -2005,7 +2014,11 @@ R_API int r_core_config_init(RCore *core) {
 
 	SETPREF("dbg.bpinmaps", "true", "Force breakpoints to be inside a valid map");
 	SETCB("dbg.forks", "false", &cb_dbg_forks, "Stop execution if fork() is done (see dbg.threads)");
-	SETCB("dbg.btalgo", "fuzzy", &cb_dbg_btalgo, "Select backtrace algorithm");
+	n = NODECB("dbg.btalgo", "fuzzy", &cb_dbg_btalgo);
+	SETDESC(n, "Select backtrace algorithm");
+	SETOPTION(n, "default");
```

Comment:
what about using vararg here like this: `SETOPTIONS(n, "default, "fuzzy", "anal", NULL)`